src/middlewared/middlewared/alembic/versions/12.0/2020-04-16_07-59_truecommand.py

```diff
@@ -0,0 +1,38 @@
+"""
+TrueCommand Service Model
+
+Revision ID: 58339783792c
+Revises: 171f5b91c36e
+Create Date: 2020-04-06 07:59:08.663553+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import middlewared.sqlalchemy as m_sa
+
+
+revision = '58339783792c'
+down_revision = '171f5b91c36e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        'system_truecommand',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('api_key', m_sa.EncryptedText(), nullable=True),
```

Comment:
If you regenerate this migration, it will yield `sa.Text()` here, why have you decided to change it like this?